tests/e2e/utils/kubeflow_installation.py

```diff
@@ -0,0 +1,200 @@
+from pickle import FALSE
+from e2e.utils.utils import load_yaml_file, print_banner
+import argparse
+from e2e.utils.utils import (
+    kubectl_wait_pods,
+    apply_kustomize,
+    install_helm,
+)
+import subprocess
+import os
+
+INSTALLATION_PATH_FILE_VANILLA = "./resources/installation_config/vanilla.yaml"
+INSTALLATION_PATH_FILE_COGNITO = "./resources/installation_config/cognito.yaml"
+
+Install_Sequence = [
```

Comment:
I see that you have to maintain this list separately because the installation sequence cannot be determined from the current config. Does it help if we change the config to a list instead of individual separated components?
e.g.
```
components:
  - cert-manager:
       paths&validation
  - istio:
```

Reply:
also INSTALL_SEQUENCE

Reply:
Are you trying to say to create another dict to maintain the install sequence?

Reply:
no, I mean if we modify the `installation_config/cognito.yaml` be to like

```
components:
  - cert-manager:
       <paths&validation>
  - istio:
```

instead of
```
cert-manager:
     <paths&validation>
istio:
```

is the order preserved when the yaml file is loaded so this dictionary can be removed?

Reply:
Ignore this as discussed